lvgl_tft/Kconfig

```
@@ -1023,30 +996,33 @@ menu "LVGL TFT Display controller"
             help
                 Configure the display BCLK (LED) pin here.
 
-        config LV_DISP_PIN_SDA
-            int "GPIO for I2C SDA" if LV_TFT_DISPLAY_PROTOCOL_I2C
-            range 0 39 if IDF_TARGET_ESP32
-            range 0 43 if IDF_TARGET_ESP32S2
-            range 0 21 if IDF_TARGET_ESP32C3
+    endmenu
 
-            default 5 if LV_PREDEFINED_DISPLAY_WEMOS_LOLIN
-            default 5
+    choice
+        prompt "Select an I2C port for the display"
+        default LV_I2C_DISPLAY_PORT_0
+        depends on LV_I2C_DISPLAY
 
-            help
-            Configure the I2C SDA pin here.
+        config LV_I2C_DISPLAY_PORT_0
+            bool
+            prompt "I2C port 0"
 
-        config LV_DISP_PIN_SCL
-            int "GPIO for I2C SCL" if LV_TFT_DISPLAY_PROTOCOL_I2C
-            range 0 39 if IDF_TARGET_ESP32
-            range 0 43 if IDF_TARGET_ESP32S2
-            range 0 21 if IDF_TARGET_ESP32C3
+        config LV_I2C_DISPLAY_PORT_1
+            bool
+            prompt "I2C port 1"
```

Provide help about I2C ports
```suggestion
            prompt "I2C port 1"
            help
                I2C is shared peripheral managed by I2C_Manager. In order to configure I2C_manager (pinout, etc.) see menu
                Component config->LVGL ESP Drivers->I2C Port Settings.
```